plugins/inputs/win_wmi/win_wmi.go

```diff
@@ -0,0 +1,268 @@
+//go:build windows
+// +build windows
+
+package win_wmi
+
+import (
+	_ "embed"
+	"fmt"
+	"runtime"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	ole "github.com/go-ole/go-ole"
+	"github.com/go-ole/go-ole/oleutil"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/filter"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+// Query struct
+type Query struct {
+	Query                string
+	Namespace            string   `toml:"Namespace"`
+	ClassName            string   `toml:"ClassName"`
+	Properties           []string `toml:"Properties"`
+	Filter               string   `toml:"Filter"`
+	TagPropertiesInclude []string `toml:"TagPropertiesInclude"`
```

Comment:
Config file property names are usually lowercase:
```suggestion
	Namespace            string   `toml:"namespace"`
	ClassName            string   `toml:"class_name"`
	Properties           []string `toml:"properties"`
	Filter               string   `toml:"filter"`
	TagPropertiesInclude []string `toml:"tag_propertes"`
```

Reply:
This is done.